proposed/http-message.md

```diff
@@ -1001,7 +1001,7 @@ interface ServerRequestInterface extends RequestInterface
      * immutability of the message, and MUST return an instance that has the
      * updated cookie values.
      *
-     * @param array $cookies Array of key/value pairs representing cookies.
+     * @param scalar[] $cookies Array of key/value pairs representing cookies.
```

Comment:
Based on the behavior of `http_parse_cookie()`, it's possible for individual cookie values to potentially be an array, IIRC. This should likely be `mixed[]`.